### PR TITLE
Fixed pytest error - name of a class that can be detected as test

### LINKED ISSUE #312: Pytest fails because of TestMapper class name (it is detected as Test class)
**Describe the bug**
TestMapper class from test_action_mapper.py is detected as Test class wronly

**To Reproduce**

1. Run pytest
2. See that it fails at pytest step

**Expected behavior**
It won't fail

**Additional context**
Seems something changed in pytest behaviour.

### PR DESCRIPTION
**Pull request by potiuk**
Fixes #312

Make sure you have checked _all_ steps below and that above description is correct.

### JIRA
- [x ] My PR addresses the following [Issue](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/312)
issue and references it in commit message.
### Tests
No need
### Commits
- [x] My commits reference the issue in the message using: Fixes #XXX /Closes #XXX or similar

### Documentation
No need

### Context

It was about test accessing non-initialized airflow DB

### CODE REVIEW

**Comment by potiuk** [comment]
Pytest fixed in this latest version - I needed to initialize airflow db before tests so that the test that accesses the DB works.

File: tests/mappers/test_action_mapper.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 TEST_DAG_NAME = "dag_name"
 
 
-class TestMapper(ActionMapper):
+class AMapper(ActionMapper):
```
Comment (sprzedwojski) [inline tests/mappers/test_action_mapper.py]:
```suggestion
class ActionMapperTestImpl(ActionMapper):
```
or similar, somehow `AMapper` bothers me.

Comment (sprzedwojski) [reply tests/mappers/test_action_mapper.py]: Or can there be no `Test` word anywhere in the name? Not even in the middle?

Comment (potiuk) [reply tests/mappers/test_action_mapper.py]: Yeah. Will change. It's Test prefix in class name (and class has to be in test_*.py or *_test.py in this case):

Pytest has several criteria to determine if class is a Test class. From https://docs.pytest.org/en/latest/goodpractices.html#conventions-for-python-test-discovery

```
pytest implements the following standard test discovery:

If no arguments are specified then collection starts from testpaths (if configured) or the current directory. Alternatively, command line arguments can be used in any combination of directories, file names or node ids.
Recurse into directories, unless they match norecursedirs.
In those directories, search for test_*.py or *_test.py files, imported by their test package name.
From those files, collect test items:
test prefixed test functions or methods outside of class
test prefixed test functions or methods inside Test prefixed test classes (without an __init__ method)
For examples of how to customize your test discovery Changing standard (Python) test discovery.

Within Python modules, pytest also discovers tests using the standard unittest.TestCase subclassing technique.


```

**Comment by potiuk** [comment]
@sprzedwojski  -> please merge when CI finishes (I go to the meeting now) :). Otherwise PRs will keep on failing.

**Comment by potiuk** [comment]
OK. Tests succeeded, so feel free to review/merge @sprzedwojski @nuclearpinguin

**Pull request merged**